build.gradle

```diff
@@ -28,7 +28,7 @@ apply from: "$rootDir/gradle-scripts/test-logger.gradle"
 
 
 ext {
-    commercetoolsJvmSdkVersion =  '1.53.0'
+    commercetoolsJvmSdkVersion =  '1.54.0'
```

Comment:
can you handle merging with the https://github.com/commercetools/commercetools-project-sync/pull/178

Reply:
Done

Reply:
I meant closing the PR with merging, so you need to handle other PR.

Reply:
Sure